Formula/wsk.rb

```diff
@@ -0,0 +1,21 @@
+class Wsk < Formula
+  desc "OpenWhisk CLI"
+  homepage "https://console.ng.bluemix.net/openwhisk/cli"
+  url "https://openwhisk.ng.bluemix.net/cli/go/download/mac/amd64/OpenWhisk_CLI-mac.zip"
+  version "0.1.0"
+
+  def install
+    bin.install "wsk"
+  end
+
+  def caveats
+    <<-EOS.undent
+      Until OpenWhisk project does not provide an official version for the CLI tools,
+      to upgrade we must uninstall/install the formula.
+    EOS
+  end
+
+  test do
+    system "true"
```

Comment:
I thought about that, but there's no `--version` or similar available from the CLI tool. Actually, there seems to be no official version at all for the OpenWhisk project: no Git tags, no (CHANGELOG|VERSION) file. Invoking the binary alone prints help to stdout and returns 0, I guess it's better then nothing